Content.Server/Medical/BiomassReclaimer/BiomassReclaimerSystem.cs

```diff
@@ -109,13 +107,10 @@ public override void Initialize()
 
         private void OnSuicide(Entity<BiomassReclaimerComponent> ent, ref SuicideEvent args)
         {
-            if (args.Handled)
-                return;
-
-            if (HasComp<ActiveBiomassReclaimerComponent>(ent))
-                return;
-
-            if (TryComp<ApcPowerReceiverComponent>(ent, out var power) && !power.Powered)
+            if (args.Handled
+                || HasComp<ActiveBiomassReclaimerComponent>(ent)
+                || TryComp<ApcPowerReceiverComponent>(ent, out var power)
+                && !power.Powered)
```

Comment:
```suggestion
                || TryComp<ApcPowerReceiverComponent>(ent, out var power) && !power.Powered)
```
This is more clear as to the function.